src/lib/components/wizard.component.ts

```diff
@@ -186,7 +196,7 @@ export class WizardComponent implements OnChanges, AfterContentInit {
     this.navigation.reset();
   }
 
-  protected updateNavigationMode(navigationModeInput: NavigationModeInput) {
-    this.model.updateNavigationMode(navigationModeFactory(this, navigationModeInput));
+  public updateNavigationMode(navigationModeInput: NavigationModeInput) {
```

Comment:
Please add some documentation to the method

Reply:
Done in  aeccff6